### DISCUSSION TITLE
tensorrt转换，flow模块推理结果不正确

### DISCUSSION THREAD

**Discussion by Zha-JiChi** [fishaudio/Bert-VITS2#384] [Q&A]
有人转换过tensorrt吗？我按照工程先转为6个onnx模型，再转为tensorrt，一共6个模型。但是使用flow模型推理时结果不正确。有人是过吗？

**Comment by Sunnycl** [comment]
请问最后解决了吗,我是导出trt后发现无法生成有效音频,定位的时候发现好像转出为ONNX格式的生成也无法生成有效音频
